DocUI/index.ts

```diff
@@ -0,0 +1,27 @@
+import { promises as fs } from "fs"
+
+import { AzureFunction, Context } from "@azure/functions"
+import { getAbsoluteFSPath } from "swagger-ui-dist"
+
+const swaggerJsdocUI: AzureFunction = async function (context: Context): Promise<void> {
+  let filename = context.bindingData.filename
+  context.log("FILE: " + filename)
+  filename = filename || "index.html"
+  context.log("FILE2: " + filename)
+  const body = await fs.readFile(getAbsoluteFSPath() + "/" + filename).then((file) => {
+    if (filename === "swagger-initializer.js") {
+      return file
+        .toString()
+        .replace('url: "https://petstore.swagger.io/v2/swagger.json"', 'url: window.origin + "/api/Doc"')
```

Comment:
this should be configurable within the swagger config, shouldn't it? [JSDoc.SwaggerDefinition](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/swagger-jsdoc/index.d.ts#L316-L327) has a `host` and `basePath` key that might do the trick to tell swagger which url to set.

just remember that we're running in multiple environments, so the url shouldn't be hard coded.

Reply:
It is needed because it is url to the swagger.json so this is the only way i found to do it. But we can discuss for any other work around.